src/PayU/Alu/Card.php

```diff
@@ -35,21 +35,41 @@ class Card
 
     /** @var bool */
     private $enableTokenCreation;
+    /**
+     * @var int|null
+     */
+    private $timeSpentTypingNumber;
+
+    /**
+     * @var int|null
+     */
+    private $timeSpentTypingOwner;
```

Comment:
I think property and getter should use `OwnerName` instead of `Owner`, to have it in sync with `cardOwnerName` property